ruby-challenges.rb

```diff
@@ -10,6 +10,14 @@
 num3 = 221
 # Expected output: '221 is odd'
 
```

Comment:
Great to be confident in your coding. It is responsible to check the outputs of the methods. Please remember to include method calls with the test variables